tests/bot/exts/filters/test_token_remover.py

```diff
@@ -283,26 +289,38 @@ def test_format_log_message(self, log_message, decoded_log_message):
             hmac="x" * len(token.hmac),
         )
 
-    @autospec("bot.exts.filters.token_remover", "LOG_MESSAGE", "USER_TOKEN_MESSAGE")
-    def test_format_log_message_user_token(self, log_message, user_token_message):
+    @autospec("bot.exts.filters.token_remover", "DECODED_LOG_MESSAGE")
+    def test_format_userid_log_message_bot(self, decoded_log_message):
+        """
+        Should correctly format the user ID portion of the log message when the user ID is
+         not found in the server.
+        """
+        token = Token("NDcyMjY1OTQzMDYyNDEzMzMy", "XsySD_", "s45jqDV_Iisn-symw0yDRrk_jf4")
+        decoded_log_message.format.return_value = " Partner"
+        msg = MockMessage(id=555, content="hello world")
+        msg.guild.get_member = MagicMock(return_value=None)
+
+        return_value = TokenRemover.format_userid_log_message(msg, token)
+
+        self.assertEqual(
+            return_value,
+            (decoded_log_message.format.return_value, False),
+        )
+        decoded_log_message.format.assert_called_once_with(
+            user_id=472265943062413332,
+        )
```

Comment:
These can each fit on one line, unless you think they are more readable when broken up.